src/components/activity-introduction/footer.tsx

```diff
@@ -1,39 +1,36 @@
 import React from "react";
+import { ProjectTypes } from "../../utilities/project-utils";
+import { renderHTML } from "../../utilities/render-html";
 
 import "./footer.scss";
 
-function Footer() {
-
-  const concordURL = "https://concord.org/";
-  const creativeCommonsURL = "https://creativecommons.org/licenses/by/4.0/";
-  const openSourceBSDURL = "http://opensource.org/licenses/BSD-2-Clause";
-  const openSourceMITURL = "http://opensource.org/licenses/MIT";
-  const openSourceApacheURL = "http://opensource.org/licenses/Apache-2.0";
+interface IProps {
+  fullWidth?: boolean;
+  projectId: number | null;
+}
 
-  const concordLink = <a href={concordURL} title="The Concord Consortium" rel="noreferrer" target="_blank">The Concord Consortium</a>;
-  const concordLink2 = <a href={concordURL} title="The Concord Consortium" rel="noreferrer" target="_blank">http://concord.org</a>;
-  const creativeCommonsLink = <a href={creativeCommonsURL} rel="noreferrer" target="_blank">Creative Commons Attribution 4.0 License</a>;
-  const openSourceBSDLink = <a href={openSourceBSDURL} rel="noreferrer" target="_blank">Simplified BSD</a>;
-  const openSourceMITLink = <a href={openSourceMITURL} rel="noreferrer" target="_blank">MIT</a>;
-  const openSourceApacheLink = <a href={openSourceApacheURL} rel="noreferrer" target="_blank">Apache 2.0</a>;
+export class Footer extends React.PureComponent<IProps> {
+  render() {
+    const { fullWidth, projectId } = this.props;
+    const projectType = ProjectTypes.find(pt => pt.id === projectId);
+    const footer = projectType?.footer || "";
+    const footerLogos = projectType?.footerLogo || [];
 
-  return (
-    <div className="footer" data-cy="footer">
-      <span>{"Copyright © 2020 "}</span>
-      {concordLink}
-      <span>{". All rights reserved. This material is licensed under a "}</span>
-      {creativeCommonsLink}
-      <span>{". The software is licensed under "}</span>
-      {openSourceBSDLink}
-      <span>{", "}</span>
-      {openSourceMITLink}
-      <span>{" or "}</span>
-      {openSourceApacheLink}
-      <span>{" licenses. Please provide attribution to the Concord Consortium and the URL "}</span>
-      {concordLink2}
-      <span>{"."}</span>
-    </div>
-  );
+    return (
+      <div className="footer" data-cy="footer">
+        <div className={`inner ${fullWidth ? "full" : ""}`}>
+          <div className={`footerText`}>
+            {renderHTML(footer)}
+          </div>
+          <div className="footer-logo-container">
+            {footerLogos.length > 0 ? footerLogos.map((logo: any, i: number) => 
+              <img key={i} src={logo} className="footer-logo" data-cy="partner-logo" />
```

Comment:
You probably want to add some margins on the footer-logo class so that the logos son't get jammed together.  It looks like this on LARA:
![image](https://user-images.githubusercontent.com/5126913/88714202-4ace5580-d0d1-11ea-96d4-987729f1cb63.png)

Reply:
Which also brings up the question of if there should be a background color on the footer to deal with images that have transparency looking odd.  Check out these HAS logos.  The transparency doesn't look great without a background:
https://activity-player.concord.org/branch/update-header-footer/?activity=sample-activity-has

Reply:
yea I didn't work on styling because the icons where appearing all over the footer (float left, right, bottom, inline) and of varying sizes(some projects had them bigger) without asking somebody. But after chatting with Leslie, I think I'll just style them to appear on the bottom and all the same height.

Reply:
Sounds good.  I would stick a margin around them to so they're not all jammed together,